api/Tests/ApiTests/DashboardPairResolvers/AuthedAdminResolverTests.swift

```diff
@@ -405,15 +419,61 @@ final class AuthedAdminResolverTests: ApiTestCase, @unchecked Sendable {
     ])
   }
 
-  func testGetAdminKeychains() async throws {
+  func testGetAdminKeychain() async throws {
     let admin = try await self.admin().withKeychain()
-    let listOutput = try await GetAdminKeychains.resolve(in: context(admin))
-    let singleOutput = try await GetAdminKeychain.resolve(
+    let output = try await GetAdminKeychain.resolve(
       with: admin.keychain.id,
       in: context(admin)
     )
+    let expected = GetAdminKeychain.Output(
+      summary: KeychainSummary(
+        id: admin.keychain.id,
+        parentId: admin.id,
+        name: admin.keychain.name,
+        description: admin.keychain.description,
+        isPublic: admin.keychain.isPublic,
+        numKeys: 1
+      ),
+      keys: [.init(from: admin.key, keychainId: admin.keychain.id)]
+    )
+
+    expect(output).toEqual(expected)
+  }
+
+  func testGetAdminKeychains() async throws {
+    let admin = try await self.admin().withKeychain()
+
+    // child with keychain assigned
+    let littleJimmy = try await self.db.create(
+      User(
+        id: .from("00000000-0000-0000-0000-000000000000"), parentId: admin.model.id,
+        name: "Little Jimmy"
+      )
+    )
+    try await self.db.create(
+      UserKeychain(childId: littleJimmy.id, keychainId: admin.keychain.id)
+    )
 
-    let expected = GetAdminKeychains.AdminKeychain(
+    // child without keychain assigned
+    let sally = try await self.db.create(
+      User(
+        id: .from("11111111-1111-1111-1111-111111111111"), parentId: admin.model.id,
+        name: "Sally"
+      )
+    )
+
+    let fromResolver = try await GetAdminKeychains.resolve(in: context(admin))
```

Comment:
only a nitpick, but i usually call these variables `output` because thats sort of the language of pairql